### PR TITLE
Install the /usr/local/bin/ccslog utility

### PR DESCRIPTION
**Pull request by glennmorris**
Adds a ccs utility script in /usr/local/bin

### CODE REVIEW

**Comment by jhoblitt** [comment]
Do you want to try to add a unit or acceptance test as a learning exercise?

**Comment by glennmorris** [comment]
Added test in f664cf5

File: spec/acceptance/basic_spec.rb [hunk 1]
```diff
@@ -117,6 +117,13 @@ class { 'ccs_software':
       it { is_expected.to be_mode '755' } # serverspec does not like a leading 0
       its(:content) { is_expected.to match %r{archive_flag=n} }
     end
+
+    describe file('/usr/local/bin/ccslog') do
+      it { is_expected.to be_file }
+      it { is_expected.to be_owned_by 'root' }
+      it { is_expected.to be_grouped_into 'root' }
+      it { is_expected.to be_mode '755' }
+    end
```
Comment (jhoblitt) [inline spec/acceptance/basic_spec.rb:126]: :+1:

**Review by jhoblitt — APPROVED** [review]
This LGTM.  I manually ran `shellcheck` on the new script.  We should probably add a shellcheck test to CI similar to lsst-it/lsst-itconfig at some point.

**Pull request merged**